Library/OcFileLib/FileProtocol.c

```diff
@@ -31,9 +31,9 @@ EfiGetEpochDays (
   IN  EFI_TIME  *Time
   )
 {
-  UINTN a;
+  UINT8 a;
   UINTN y;
-  UINTN m;
+  UINT8 m;
```

Comment:
Are these changes trying to reduce stack usage or fixing any potential security problem?
* If it is the former, UINTN alignment will not let it become less, will have to reorder the variable declaration.
* If it is the latter, I do not get the reasoning behind either.

Reply:
Imo walk the whole way and also change the other variables away from UINTN (and potentially rearrange), or don't bother at all